edx_when/api.py

```diff
@@ -24,19 +24,25 @@ def _ensure_key(key_class, key_obj):
     return key_obj
 
 
-def is_enabled_for_course(course_key):
+def _are_relative_dates_enabled(course_key=None):
     """
-    Return whether edx-when is enabled for this course.
+    Return whether it's OK to consider relative dates. If not, pretend those database entries don't exist.
     """
-    return models.ContentDate.objects.filter(course_id=course_key, active=True).exists()
+    try:
+        # It's bad form to depend on LMS code from inside a plugin like this. But we gracefully fail, and this is
+        # temporary code anyway, while we develop this feature.
+        from lms.djangoapps.courseware.tabs import ENABLE_DATES_TAB
```

Comment:
I can see where we might want to rename this if using it here too.

Reply:
Yeah... I wanna chat about this before landing. Maybe a new waffle flag that isn't tied to a tab feature?  Or reuse-but-rename?

Reply:
Done. Using a new flag now.